data-generation/src/main/scala/thesis/Landy.scala

```diff
@@ -12,28 +12,45 @@ import scala.collection.mutable
 import scala.math.Ordered.orderingToOrdered
 
 
-class Landy(graph: LandyAttributeGraph) extends TemporalGraph[LandyEntityPayload, LandyEntityPayload] {
-  override val vertices: VertexRDD[LandyEntityPayload] = graph.vertices
-  override val edges: EdgeRDD[LandyEntityPayload] = graph.edges
-  override val triplets: RDD[EdgeTriplet[LandyEntityPayload, LandyEntityPayload]] = graph.triplets
+class Landy(graph: LandyAttributeGraph) extends TemporalGraph {
+  val vertices: VertexRDD[LandyEntityPayload] = graph.vertices
+  val edges: EdgeRDD[LandyEntityPayload] = graph.edges
 
-  override def snapshotAtTime(instant: Instant): Graph[LandyEntityPayload, LandyEntityPayload] = {
+  def snapshotAtTimeLandy(instant: Instant): Graph[LandyEntityPayload, LandyEntityPayload] = {
     val vertices = localVertices
       .filter(vertex =>
         vertex._2.validFrom < instant &&
           vertex._2.validTo >= instant
       )
       .map(vertex => (vertex._2.id, vertex._2))
 
-    val edges = this.edges.filter(edge =>
+    val edges = this.graph.edges.filter(edge =>
```

Comment:
Hvorfor this.graph.edges her? Det er kanskje bedre å gjøre det sånn som dette enn å definere hva `vertices` og `edges` er lenger opp i fila?

Reply:
I fremtiden tenker jeg egentlig at TemporalGraph.{vertices,edges} returnerer noder og kanter hvordan de ser ut akkurat når det blir hentet. TemporalGraph.vertices == Temporalgraph.snapshotAtTime(Instant.now).
Graph-feltet i landy burde nok egentlig het "underlayingGraph" for å ikke blande konseptene

Reply:
> hvordan de ser ut akkurat når de blir hentet

Ja, enig! Siden `vertices` og `edges` på SnapshotDelta-objektet gir ikke så veldig mye mening for SnapshotDelta atm.

> Graph-feltet i landy burde nok egentlig het "underlyingGraph"

Støttes!